tests/conftest.py

```diff
@@ -62,6 +66,18 @@ def metadata_schema_valid(self) -> bool:
 
         return True
 
+    def copy_input_fixtures(self, mock_data_path: str):
+        shutil.copytree(mock_data_path, self.input_dir, dirs_exist_ok=True)
+
+    def assert_result_snapshots(self):
+        for result_file in self.result_files():
```

Comment:
How does this guard against asserting just omitting some files from the result set?